t/re/pat.t

```diff
@@ -2512,6 +2512,94 @@ SKIP:
         ok($str =~ s/$copy/PQR/, 'replaced $copy with PQR');
         is($str, "PQR", 'final string should be PQR');
     }
+
+
+    # Various tests for regexes with code blocks interpolated from an
```

Comment:
A minor point: all these tests involve regexps that match, having some that correctly fail to match would be good.